plugins/colordialog/dialogs/colordialog.js

```diff
@@ -119,11 +119,11 @@ CKEDITOR.dialog.add( 'colordialog', function( editor ) {
 	function setHighlight( color ) {
 		if ( color ) {
 			$doc.getById( hicolorId ).setStyle( 'background-color', color );
-			$doc.getById( hicolorTextId ).setHtml( color );
+			$doc.getById( hicolorTextId ).setHtml( CKEDITOR.tools.htmlSafeByReview(color, 'The variable color here is only ever set to the html of elements created internally (in createColorTable and applyColorCell)') );
```

Comment:
Can color be sanitized? Let's try to see if it breaks things when we sanitize it, if so we can legacy convert or see if we can lax sanitize somehow.

Reply:
Yep, looks like color can be htmlEncoded without any breakages.